### PR TITLE
certdump: unstable-2023-07-12 -> 0-unstable-2023-12-25

### PR DESCRIPTION
**Pull request by JamieMagee**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by JamieMagee** [comment]
```
$ curl -L -O "https://zoom.us/client/6.2.6.49050/ZoomInstallerFull.exe?archType=x64"
$ ./result/bin/CertDump -f ZoomInstallerFull.exe
$ openssl pkcs7 -inform der -in cert.dump -print_certs
```

<details>
<summary>Details</summary>

```
subject=C=US, O=DigiCert Inc, OU=www.digicert.com, CN=DigiCert Trusted Root G4
issuer=C=US, O=DigiCert Inc, OU=www.digicert.com, CN=DigiCert High Assurance EV Root CA
-----BEGIN CERTIFICATE-----
MIIFwDCCBKigAwIBAgIQD9G7ynlr1/jdTILhCpqWMTANBgkqhkiG9w0BAQsFADBs
MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3
d3cuZGlnaWNlcnQuY29tMSswKQYDVQQDEyJEaWdpQ2VydCBIaWdoIEFzc3VyYW5j
ZSBFViBSb290IENBMB4XDTIyMDExMzAwMDAwMFoXDTMxMTEwOTIzNTk1OVowYjEL
MAkGA1UEBhMCVVMxFTATBgNVBAoTDERpZ2lDZXJ0IEluYzEZMBcGA1UECxMQd3d3
LmRpZ2ljZXJ0LmNvbTEhMB8GA1UEAxMYRGlnaUNlcnQgVHJ1c3RlZCBSb290IEc0
MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAv+aQc2jeu+RdSjwwIjBp
M+zCpyUuySE98orYWcLhKac9WKt2ms2uexuEDcQwH/MbpDgW61bGl20dq7J58soR
0uRf1gU8Ug9SH8aeFaV+vp+pVxZZVXKvaJNwwrK6dZlqczKU0RBEEC7fgvMHhOZ0
O21x4i0MG+4g1ckgHWMpLc7sXk7Ik/ghYZs06wXGXuxbGrzryc/NrDRAX7F6Zu53
yEioZldXn1RYjgwrt0+nMNlW7sp7XeOtyU9e5TXnMcvak17cjo+A2raRmECQecN4
x7axxLVqGDgDEI3Y1DekLgV9iPWCPhCRcKtVgkEy19sEcypukQF8IUzUvK4bA3Vd
eGbZOjFEmjNAvwjXWkmkwuapoGfdpCe8oU85tRFYF/ckXEaPZPfBaYh2mHY9WV1C
doeJl2l6SPDgohIbZpp0yt5LHucOY67m1O+SkjqePdwA5EUlibaaRBkrfsCUtNJh
besz2cXfSwQAzH0clcOP9yGyshG3u3/y1YxwLEFgqrFjGESVGnZifvaAsPvoZKYz
0YkH4b235kOkGLimdwHhD5QMIR2yVCkliWzlDlJRR3S+Jqy2QXXeeqxfjT/JvNNB
ERJb5RBQ6zHFynIWIgnffEx1P2PsIV/EIFFrb7GrhotPwtZFX50g/KEexcCPorF+
CiaZ9eRpL5gdLfXZqbId5RsCAwEAAaOCAWYwggFiMA8GA1UdEwEB/wQFMAMBAf8w
HQYDVR0OBBYEFOzX44LScV1kTN8uZz/nupiuHA9PMB8GA1UdIwQYMBaAFLE+w2kD
+L9HAdSYJhoIAu9jZCvDMA4GA1UdDwEB/wQEAwIBhjATBgNVHSUEDDAKBggrBgEF
BQcDAzB/BggrBgEFBQcBAQRzMHEwJAYIKwYBBQUHMAGGGGh0dHA6Ly9vY3NwLmRp
Z2ljZXJ0LmNvbTBJBggrBgEFBQcwAoY9aHR0cDovL2NhY2VydHMuZGlnaWNlcnQu
Y29tL0RpZ2lDZXJ0SGlnaEFzc3VyYW5jZUVWUm9vdENBLmNydDBLBgNVHR8ERDBC
MECgPqA8hjpodHRwOi8vY3JsMy5kaWdpY2VydC5jb20vRGlnaUNlcnRIaWdoQXNz
dXJhbmNlRVZSb290Q0EuY3JsMBwGA1UdIAQVMBMwBwYFZ4EMAQMwCAYGZ4EMAQQB
MA0GCSqGSIb3DQEBCwUAA4IBAQBB8akR9A8TR3nJZWqNwO6YBKTG4wzyorHnX2JP
2pNZNs3yHuEe7A/hGz+R1OQPwuaYNP8ZD2YoH09iMaQq4SoTutc+x9GS/qvd36qY
M0YfTp694O0HRuuALqT9rlO0pMGk9SGNa8wBrfUZ7teQpTJSK9vMcw8Y+PvO3oNx
p++7UVsEdi+trdPqHfz9U5PljInjYBm54C8jjixcSct+K4g5K7kGYYjzrgJ2dnTB
ZJG4iVGX5YV9PdNXplhuH+77aWHiWtwoIehjuj7PVu2a5PcF6srz7BLfOM3aez4r
k1MvXFEAWN6QSKMKxwv0O21VMfACzEd3llGjXrW8XMhDEaGn
-----END CERTIFICATE-----

subject=C=US, O=DigiCert, Inc., CN=DigiCert Trusted G4 Code Signing RSA4096 SHA256 2021 CA1
issuer=C=US, O=DigiCert Inc, OU=www.digicert.com, CN=DigiCert Trusted Root G4
-----BEGIN CERTIFICATE-----
MIIGsDCCBJigAwIBAgIQDk1n9kMWySo7ehfMRpdqjzANBgkqhkiG9w0BAQsFADBi
MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3
d3cuZGlnaWNlcnQuY29tMSEwHwYDVQQDExhEaWdpQ2VydCBUcnVzdGVkIFJvb3Qg
RzQwHhcNMjEwNzI4MDAwMDAwWhcNMzYwNzI3MjM1OTU5WjBpMQswCQYDVQQGEwJV
UzEXMBUGA1UEChMORGlnaUNlcnQsIEluYy4xQTA/BgNVBAMTOERpZ2lDZXJ0IFRy
dXN0ZWQgRzQgQ29kZSBTaWduaW5nIFJTQTQwOTYgU0hBMjU2IDIwMjEgQ0ExMIIC
IjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAri4MfdPvaOOECPqbDE3zNg/n
sfM0MoNGVGJf6ogVscpox00cZPOiAXZwGLyPm/HmSxmNCDxKUqFWE1LIVTzCYcNt
kqCv+MNCZoHxIY1mERdsU+sYYfADDDHpi10op0br0SrrrEuf6+uuXRxhpZ7xgopt
T/WhDG3vqXjabJPsKCDcEsnElut6Nm8XHHV+CcnZpbwle4OCcnrgLerBEHu+iIEj
gCts01n+1b1abmrFXzJ3aC/iXP/6HIqYs7sdAUU8C7WcLflXm3xzec6yltqyKEyx
cvgdBMTapqmw7pkFrm8l56JYpctFHJtdyFsroL6lojC8yS3j5z05+SgaAjSMI8Qf
al5WAy2Jii/EvnPZf1azgUvzYkfKgYEy5kA5c+3j9ZuKifb7EPp4pvkXHZJ6KzJh
abuRgXqjObO9YtRk5LnZgvwHJ74T/vFTYO47h2wOqv/JaZd8K5/IAiSop/JTjlHB
stkJfnAb6iH1UUKKgMhpNCpSkM9s2xzQQmZ44RGIRDs+AHm5lER4Z2OohQ+nMP/g
UjvnouZ2XZuI+yn2EBbtRDkBr9YRru28bpsKNCALgFmNzoxR7rs6FaQBzEsgDj80
u6F04YlFIQwp9U8iWTljWWMX+p8OJ7Q1LXAPQl/aSZmlf+WLXQcr8x0qU+v7Xi1T
VSBZpvRMa0s941GE0WkCAwEAAaOCAVkwggFVMBIGA1UdEwEB/wQIMAYBAf8CAQAw
HQYDVR0OBBYEFLxrImWdjGjm5h4F8dhyJdS44EjAMB8GA1UdIwQYMBaAFOzX44LS
cV1kTN8uZz/nupiuHA9PMA4GA1UdDwEB/wQEAwIBhjATBgNVHSUEDDAKBggrBgEF
BQcDAzB3BggrBgEFBQcBAQRrMGkwJAYIKwYBBQUHMAGGGGh0dHA6Ly9vY3NwLmRp
Z2ljZXJ0LmNvbTBBBggrBgEFBQcwAoY1aHR0cDovL2NhY2VydHMuZGlnaWNlcnQu
Y29tL0RpZ2lDZXJ0VHJ1c3RlZFJvb3RHNC5jcnQwQwYDVR0fBDwwOjA4oDagNIYy
aHR0cDovL2NybDMuZGlnaWNlcnQuY29tL0RpZ2lDZXJ0VHJ1c3RlZFJvb3RHNC5j
cmwwHAYDVR0gBBUwEzAHBgVngQwBAzAIBgZngQwBBAEwDQYJKoZIhvcNAQELBQAD
ggIBAEnyvKd58wdZXmcA+X7z6oYDbVHvIUfZ06j7ceH8uVBtYIDZHk5oVHGzVui4
wx3D97fdWCqsFL+B59Z9jZcTPdrIlmstIiLkystVOrkyDRsOoiI/6O/nuvxOS+iS
o+9MLMiU/rUB9YrzLL+USwFeIitwL4CRhdf+Zuar/NYsy1qU8cHV04pGIdie2mPK
fLa5TqRhA1BElbYPcakAoWKx//O1B04BYiVAbU9dEr2n+pItB2/QzjaIm9tNbcry
eXW8skjjVMp8AP6t/y9x0XchZ5pVbTABf/qVT3ffxsyzgyL7HyWgC/fVR7rbgVoX
9Bw/GmHw/xidbZdYWBceThHp1mR2MJtbA6wr3l9c+FguOMF51v9QV45/nhGCQJLc
nOgdM2wbz13Y7udRoF1pbBfDdBqyo1m9Ztl8NDyUnGp8+ujPUu+egIgPy3Z8hSCP
bLKQNSMsgRSdcrX+Kd+OOzZKpDvH2EJwzNRqizJLL+luO4AKf8NKIlTXHFbGGaXh
mNXB6PSUL2XobkwHmA9ZLV6AQ5G3S1gBMYnLBbbjEmuMYzWpT/4WWydXvGZGkCgw
MqHKVHkOp4Rxy31w8JJBVyp2NTvdr5JXNlrb1U0dJU1Ie0bNkXxvw5vVvv0jhgSL
d2cMpodm6dMlNU5TWt2QtLrUKtfNvcLKeF6wG2ayii6COsXo
-----END CERTIFICATE-----

subject=jurisdictionC=US, jurisdictionST=Delaware, businessCategory=Private Organization, serialNumber=4969967, C=US, ST=California, L=San Jose, O=Zoom Video Communications, Inc., CN=Zoom Video Communications, Inc.
issuer=C=US, O=DigiCert, Inc., CN=DigiCert Trusted G4 Code Signing RSA4096 SHA256 2021 CA1
-----BEGIN CERTIFICATE-----
MIIH4jCCBcqgAwIBAgIQBLXvc7+yUSuolAEiMT04IDANBgkqhkiG9w0BAQsFADBp
MQswCQYDVQQGEwJVUzEXMBUGA1UEChMORGlnaUNlcnQsIEluYy4xQTA/BgNVBAMT
OERpZ2lDZXJ0IFRydXN0ZWQgRzQgQ29kZSBTaWduaW5nIFJTQTQwOTYgU0hBMjU2
IDIwMjEgQ0ExMB4XDTI0MTAxMDAwMDAwMFoXDTI1MDExMzIzNTk1OVowgeoxEzAR
BgsrBgEEAYI3PAIBAxMCVVMxGTAXBgsrBgEEAYI3PAIBAhMIRGVsYXdhcmUxHTAb
BgNVBA8MFFByaXZhdGUgT3JnYW5pemF0aW9uMRAwDgYDVQQFEwc0OTY5OTY3MQsw
CQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5pYTERMA8GA1UEBxMIU2FuIEpv
c2UxKDAmBgNVBAoTH1pvb20gVmlkZW8gQ29tbXVuaWNhdGlvbnMsIEluYy4xKDAm
BgNVBAMTH1pvb20gVmlkZW8gQ29tbXVuaWNhdGlvbnMsIEluYy4wggIiMA0GCSqG
SIb3DQEBAQUAA4ICDwAwggIKAoICAQDPEqv1TP6NvmWERsW97+4N9bl8OWdcZJqK
j6ualyh39h21sOU6bejAoqEGyhl+VixevspeymnCM0szM84/X+DwlfmpynUL9jiP
V3re+/HXtEMhZffpHovmPPIloV64rXI64kL/M25iyiSgX/0uCmzI5rSWdzEdRpHw
CTOq4s7tNLeaZ7KyJVKc+9J13BGjfABJxga+XI3W84bDkaXxeaREQ8TBeeNUFAHA
9TakkeFxzSz9AmlNfqzM4Wk/vCN0Wg6UKNX813Khy8jVQaJJbOF6YBPmp59ibntk
VTVHXEzJ1neNBkOWr8wwIYGomeKdacfzYE6/crWh0WZ9CqQ5ok7cjCQOhqTf7a79
S3UeQtVR4sgCP/wgaBVBnKj6kj3FJ1QWHtXfOHg5fIVKsVMjCPeBH9oyU/tpEVg7
SIFNAb4td0YFX6tVQFc1eV5wL46PFX9x+xm50FN9NCKvJgBbuP459FRRW612740e
jFITHTPIQhxDFb1xaHJY/zBLQG+TZg86ewfWFm9bV0frr12iQpELnYpnASZWpSxO
4FzXSFhOSqlMK06Ycwt0sT3ZvlWfNSOiiWIspdf/NysnwKAXlCm4hZwQITD947Xk
zrc2yfzTzLq0HWPnQiNgQZJXIMtgTjicEmrAPsra0rlJg9XGtKIZE94BtTFeV3RR
AzMQ0EKzOwIDAQABo4ICAjCCAf4wHwYDVR0jBBgwFoAUvGsiZZ2MaObmHgXx2HIl
1LjgSMAwHQYDVR0OBBYEFMOfJ/vWY93p/T9DHebQmg8FkYm+MD0GA1UdIAQ2MDQw
MgYFZ4EMAQMwKTAnBggrBgEFBQcCARYbaHR0cDovL3d3dy5kaWdpY2VydC5jb20v
Q1BTMA4GA1UdDwEB/wQEAwIHgDATBgNVHSUEDDAKBggrBgEFBQcDAzCBtQYDVR0f
BIGtMIGqMFOgUaBPhk1odHRwOi8vY3JsMy5kaWdpY2VydC5jb20vRGlnaUNlcnRU
cnVzdGVkRzRDb2RlU2lnbmluZ1JTQTQwOTZTSEEyNTYyMDIxQ0ExLmNybDBToFGg
T4ZNaHR0cDovL2NybDQuZGlnaWNlcnQuY29tL0RpZ2lDZXJ0VHJ1c3RlZEc0Q29k
ZVNpZ25pbmdSU0E0MDk2U0hBMjU2MjAyMUNBMS5jcmwwgZQGCCsGAQUFBwEBBIGH
MIGEMCQGCCsGAQUFBzABhhhodHRwOi8vb2NzcC5kaWdpY2VydC5jb20wXAYIKwYB
BQUHMAKGUGh0dHA6Ly9jYWNlcnRzLmRpZ2ljZXJ0LmNvbS9EaWdpQ2VydFRydXN0
ZWRHNENvZGVTaWduaW5nUlNBNDA5NlNIQTI1NjIwMjFDQTEuY3J0MAkGA1UdEwQC
MAAwDQYJKoZIhvcNAQELBQADggIBAA81pmO0/I/XR3y2m5ANnWNTtRptOUzx5kde
kCItcqMPEkYXqDkTFwmgS0j2S+Z3yPs+ETuB67TK87W5NTGfpfBNODZ42gqhTc9w
g68VFceygYvPIjY0yEqiUAKVDXrJcxS3oz/A6F7KQRmcX6uisaQrDD17INEmSf5M
5F7cAoo1Z5dd32b2OAgu2vSQrTSdL8QG/DnYWtVmlKumo9pc5RSTzLwfqoyZR2sb
K2VK6/MtZ+D8tC9ffmtOJAAtNQSTIF14ntBmYDJSkXf5Sj1KB86Kj/EDxupyAfm7
w266/WLHFogqaoqVOI91UZR13EQqkchgpt5CBXV6DvQMrXPtudEfUtRapl1PNQ8I
WW6T1VJ8MTp9I20a7Pio7w5Od3TFAwWIJC0pn7ps+31NP9hfsvS7vqz3PBJLsca2
XNVBBZu+en6fg4oCDwKqHlnXqGwCDvoEPvUICtFNO3HMP+E3V7LukWhtwR1t0yIc
vtCm6F6lxwdIeyN5IVPGCJJesF2e4wbJ7IeunPt5SZgpgbLdehKZAd6LbqWKWZty
XoQ40bAJYF4IxIY7CpkgM5qEqrdcSA+13EZ6nLMLfwnyXmzTHckadl4UovUs4r8E
5I1ahChaGOWbOv+Wfi3NMShK0V9F3X9BBa+RZh+AqIlGNsAPmu4E3PqYWTdzFwBz
BwY2Gg58
-----END CERTIFICATE-----
```

</details>

File: pkgs/by-name/ce/certdump/package.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+buildDotnetModule rec {
+  pname = "certdump";
+  version = "unstable-2023-12-25";
```
Comment (kirillrdy) [inline pkgs/by-name/ce/certdump/package.nix:10]:
```suggestion
  version = "0-unstable-2023-12-25";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

File: pkgs/by-name/ce/certdump/package.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  src = fetchFromGitHub {
+    owner = "secana";
+    repo = "CertDump";
+    rev = "a834da24ee18503109631d836540a2b0cb481517";
+    sha256 = "sha256-86s6KLP0DliKOr0fVId7SGN333b7HkiL5p/q0vazwMc=";
```
Comment (kirillrdy) [inline pkgs/by-name/ce/certdump/package.nix:16]:
```suggestion
    hash = "sha256-86s6KLP0DliKOr0fVId7SGN333b7HkiL5p/q0vazwMc=";
```
https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub

**Review by baloo — APPROVED** [review]

File: pkgs/by-name/ce/certdump/package.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  meta = {
+    description = "Dump certificates from PE files in different formats";
+    mainProgram = "CertDump";
+    homepage = "https://github.com/secana/CertDump";
+    longDescription = ''
+      Cross-Platform tool to dump the signing certificate from a Portable Executable (PE) file.
+    '';
+    license = lib.licenses.asl20;
+    maintainers = builtins.attrValues { inherit (lib.maintainers) baloo; };
```
Comment (Artturin) [inline pkgs/by-name/ce/certdump/package.nix:37]: What's up with this, It probably has worse performance than `[ lib.maintainers.baloo ]`.

Comment (JamieMagee) [reply pkgs/by-name/ce/certdump/package.nix:37]: Sorry about that. Addressed it in #352705

**Pull request merged**